### ISSUE TITLE
Ludwig model with different csv

### ISSUE THREAD

**Issue by PingoPong** [ludwig-ai/ludwig#563]
I want use this example:
Natural Language Understanding¶
utterance | intent | slots
I want a pizza | order_food | O O O   B-Food_type
Book a flight to Boston | book_flight | O O O O B-City
Book a flight at 7pm to London | book_flight | O O O O   B-Departure_time O B-City

I made about 5 experiments. The first experiment with csv N°1 and the other experiments with csv N°2:
1.	csv: Example ........., order_food, O O O **PType_1 PType_2** ......
2.	csv:  Example ........., order_food, O O O **P_TypePizza P_TypeMargerita** ......
The file *.yaml is the same for 2 csv.
Both csv files have about 1.500 rows. 
When I loaded the  models from the correct path for 2 different csv files and I made the predict, in slots I look always the PType_1 PType_2… (the variables present in the first csv), even if I uploaded the model generated with the second csv file. I expected to look in the slots the P_TypePizza instead of PType_1. I’m sure that I load the model from correct path.
So I gave another name (--experiment_name) for the second csv and I do a new experiment and I load the model from the path with the correct name experiment  but don't work. 
I expected to look in the slots the P_TypePizza (second csv) instead of PType_1 (first csv).
The data seem to me mixed (data from the first csv with data from the second csv)

Where am I doing wrong?
*.yaml:
input_features:
    -
        name: utterance
        type: text
        level: word
        encoder: rnn
        cell_type: lstm
        bidirectional: true
        num_layers: 2
        reduce_output: null
        preprocessing:
          word_format: space

output_features:
    -
        name: intent
        type: category
        reduce_input: sum
        num_fc_layers: 1
        fc_size: 64
    -
        name: slots
        type: sequence
        decoder: tagger
        
training:
  batch_size: 8
  epochs: 400
  early_stop: 40
  dropout_rate: 0.6
  learning_rate: 0.0001

**Environment (please complete the following information):**
 - OS: Win 10
- Python version 3.6.6
- Ludwig version 0.2.1
![DataFrame of a predict](https://user-images.githubusercontent.com/57320566/68479898-b9ae0a80-0233-11ea-8e32-e4f0dad98d8b.png)

**Additional context**

**Comment by msaisumanth** [comment]
Please share the commands you used for training and prediction.

**Comment by z-dfalc** [comment]
For model I use:
ludwig experiment --experiment_name Dino --data_csv C:\Users\dino\AppData\Local\Programs\Python\Python36\Scripts\Bot\Pizza.csv --model_definition_file C:\Users\dino\AppData\Local\Programs\Python\Python36\Scripts\Bot\model_define.yaml

For predict I use the API:
model_dir ="C:/Users/dino/AppData/Local/Programs/Python/Python36/Scripts/results/Dino_run/model"
ludwig_model = LudwigModel.load(model_dir)

    client_w=input("Tu: ")
    test_df=ludwig_model.predict(data_df=pd.DataFrame({'utterance': [client_w]}))
Where client_w is the txt to predict.

Now I have repeated the experiment and now it works correctly.

![DataFrame of a predict_2](https://user-images.githubusercontent.com/57320566/68516612-c7d84700-0284-11ea-91f2-d7414b6c34c9.png)

But I didn't understand why it didn't work properly before.

**Comment by msaisumanth** [comment]
@z-dfalc The issue is not clear to me at all. You need to explain a little better. 

What I understand is you're training the model on the first csv and trying to predict on the second one. If that's the case, the model will predict the labels that are from the first csv. 

How would the model know "P_TypePizza" when you haven't used this data for training the model?

**Comment by w4nderlust** [comment]
Can you please show us the first few lines of the data you used for training and test?

**Comment by z-dfalc** [comment]
Sorry for my unclear explanation. 
I created various models with two different csv files.
The “utterance” was written in Italian language and is the same in the two csv files, only the value of the slots changes in the two files.
When I do the predict I found mixed values of the slots (see the first 

[Pizza_1.zip](https://github.com/uber/ludwig/files/3835117/Pizza_1.zip)

figure above) and I can't understand where I went wrong.
I also ask myself the same question.
In atthaced you can find the 2 csv files, they are simple files that I used to try and learn Ludwig. 
The phrases in csv files are repeated in blocks.
Thanks 
[Pizza_2.zip](https://github.com/uber/ludwig/files/3835119/Pizza_2.zip)
